src/cli.rs

```diff
@@ -74,6 +77,12 @@ impl Options {
                 .multiple(true)
                 .conflicts_with("q")
                 .help("Increases the level of verbosity (the max level is -vvv)"))
+            .arg(Arg::with_name("command")
+                .short("e")
+                .multiple(true)
+                .takes_value(true)
+                .allow_hyphen_values(true)
+                .help("Command and args to execute (must be last argument)"))
```

Comment:
If we take a single string, then we have to parse that string. It's way easier to just say it should be the last argument.